src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -73,7 +74,8 @@ public TableValue(TableType type)
         internal TableValue(IRContext irContext)
             : base(irContext)
         {
-            Contract.Assert(IRContext.ResultType is TableType);
```

Comment:
>TableType [](http://example.com/codeflow?start=52&length=9)

Why this change?

Reply:
To allow ConnectorTableType which is not a TableType (sealed)

Reply:
Can it be? 

Else, this may end up breaking other code that does this kind of check.

Reply:
I imagined there was a good reason
I'll "unseal" TableType and update tormoow